src/NotepadNext/dialogs/MainWindow.cpp

```diff
@@ -507,8 +507,8 @@ MainWindow::MainWindow(NotepadNextApplication *app) :
 
     ui->actionAboutNotepadNext->setShortcut(QKeySequence::HelpContents);
     connect(ui->actionAboutNotepadNext, &QAction::triggered, [=]() {
-        QMessageBox::about(this, "About Notepad Next",
-                            QString("<h3>Notepad Next v%1</h3>"
+        QMessageBox::about(this, tr("About Notepad Next"),
+                            tr("<h3>Notepad Next v%1</h3>"
```

Comment:
This string should not be translated.

Reply:
What about Chinese and other languages that use different alphabets?

Reply:
So actually my question is that why should we leave it in English? I misunderstood something, just forget the different alphabet. :)

Reply:
This call to `tr()` covers the entire block of text in the about dialog including the verbiage regarding the copyright ownership and the license itself.

I would not feel comfortable changing the legal terminology. The Free Software Foundation also has [no official translations](https://www.gnu.org/licenses/translations.en.html) of the license. So I feel its best to keep this in English for legal reasons.

Reply:
I see, thank you for the clarification!